### PR TITLE
Added compression

### PR DESCRIPTION
**Pull request by george-angel**
Reduce the size of strongbox files

### CODE REVIEW

File: app.go [hunk 1]
```diff
@@ -202,6 +205,34 @@ func encrypt(b []byte, pub, priv [32]byte) ([]byte, error) {
...
+	}
+	return buf.Bytes()
+}
+
+func decompress(b []byte) []byte {
+	zr, err := gzip.NewReader(bytes.NewReader(b))
+	if err != nil {
+		log.Fatal(err)
+	}
+	zr.Close()
```
Comment (catalin-ilea) [inline app.go]: You need a defer here

Comment (catalin-ilea) [inline app.go]: "Need" is a strong word as zr.Close() does nothing in this instance but still best practice have it there. Feel free to ignore.

**Comment by mjgarton** [comment]
LGTM.  Now that we're layering things, it feels slightly less nice that we're not using io.Reader and io.Writer any more, but that's fine for now I think. (until/unless me or someone makes a PR for that)

**Comment by george-angel** [comment]
@mjgarton Agreed, as you said that can be a separate PR to do with general structure.

**Pull request merged**